master/usage/removing-nodes.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Removing Nodes
+---
+
+With each Calico node there is a Node entry in the datastore and Workload
+Endpoints are associated with that node.  As nodes are replaced (with
+differently named nodes) or decomissioned, the datastore will accumulate
```

Comment:
I would argue that replacing a node with a differently named node is decommissioning the first node (since we don't have the concept of "replacing a node".